pkg/cable/driver.go

```diff
@@ -53,7 +54,14 @@ func AddDriver(name string, driverCreate DriverCreateFunc) {
 func NewDriver(localEndpoint types.SubmarinerEndpoint, localCluster types.SubmarinerCluster) (Driver, error) {
 	driverCreate, ok := drivers[localEndpoint.Spec.Backend]
 	if !ok {
-		return nil, fmt.Errorf("unsupported cable type %s", localEndpoint.Spec.Backend)
+		var driverList strings.Builder
+		for driver := range drivers {
+			if driverList.Len() > 0 {
+				driverList.WriteString(", ")
+			}
+			driverList.WriteString(driver)
+		}
```

Comment:
This is fine - I wasn't aware of `strings.Builder` - but simpler to use `strings.Join`?

Reply:
I couldn’t figure out a way to use `Join` on a map...

Reply:
ahh - sorry I thought it was an array.  It would be nice if there were built-in map utilities like Keys(), Values() etc like Java has. Maybe there is somewhere ...